frontend/components/CellInput.js

```diff
@@ -53,33 +53,50 @@ import { cell_movement_plugin, prevent_holding_a_key_from_doing_things_across_ce
 import { pluto_paste_plugin } from "./CellInput/pluto_paste_plugin.js"
 import { bracketMatching } from "./CellInput/block_matcher_plugin.js"
 import { cl } from "../common/ClassTable.js"
+import { literal, 
+    macroName,
+    std_variableName,
+    bool,
+    comment,
+    atom,
+    number,
+    keyword,
+    string,
+    variableName,
+    def_variableName,
+    bracket,
+    brace,
+    tagName,
+    link,
+    invalid,
+    invalid_bg } from "./SyntaxColorsParser.js"
 
 export const pluto_syntax_colors = HighlightStyle.define([
     /* The following three need a specific version of the julia parser, will add that later (still messing with it 😈) */
     // Symbol
-    { tag: tags.literal, color: "#5e7ad3", fontWeight: 700 },
-    { tag: tags.macroName, color: "#5668a4", fontWeight: 700 },
+    { tag: tags.literal, color: literal, fontWeight: 700 },
```

Comment:
```suggestion
    { tag: tags.literal, color: "var(--cm-builtin-color)", fontWeight: 700 },
```

It turns out you can use CSS variables here! The would simplify our code, and it means that you don't need to reload Pluto after switching to dark mode 🌝

Reply:
Good idea! So basically for all the variables imported from `SyntaxColorParser.js` are not needed, right? You would only refer to the CSS variable directly. Am I understanding this correctly?